tornettools/parse_oniontrace.py

```diff
@@ -13,7 +13,7 @@ def parse_oniontrace_logs(args):
         logging.warning("Unable to parse oniontrace simulation data.")
         return
 
-    cmd_str = f"{otracetools_exe} parse -m {args.nprocesses} -e '.*\.oniontrace\.[0-9]+.stdout' shadow.data/hosts"
+    cmd_str = f"{otracetools_exe} parse -m {args.nprocesses} -e '.*\.oniontrace\.[0-9]+.stdout' -e 'stdout.*\.oniontrace\.[0-9]+.log' shadow.data/hosts"
```

Comment:
A comment explaining why there are two patterns and which is which would be helpful.

Reply:
Done!